docs/api/csf/csf-factories.mdx

```diff
@@ -0,0 +1,328 @@
++ import { definePreview } from '@storybook/your-framework';
+- import { Preview } from '@storybook/your-renderer';
+// 👇 Import the addons you are using
++ import addonA11y from '@storybook/addon-a11y';
+
++ export default definePreview({
+- export const preview: Preview = {
+    // ...current config
+    // 👇 Add your addons here
++   addons: [addonA11y()],
++ });
+- };
+- export default preview;
+```
+
+### 4. Update your story files
+
+Story files have been updated for improved usability. With the new format:
+
+- Import the preview construct from the Storybook preview file
+- The meta object is now created via the [`preview.meta`](#previewmeta) function and does not have to be exported as a default export
+- Stories are now created from the meta object, via the [`meta.story`](#metastory) function
+
+<Callout variant="info">
+The examples below show the changes needed to upgrade a story file from CSF 3 to CSF Factories. You can also upgrade from CSF 1 or 2 using similar steps.
+</Callout>
+
+```diff title="Button.stories.ts"
+// Learn about the # subpath import: https://storybook.js.org/docs/api/csf/csf-factories#subpath-imports
++ import preview from '#.storybook/preview';
+- import { Meta, StoryObj } from '@storybook/your-renderer';
+
+import { Button } from './Button';
+
++ const meta = preview.meta({
+- const meta = {
+    // ...current meta
++ });
+- } satisfies Meta<typeof Button>;
+- export default meta;
+
+- type Story = StoryObj<typeof meta>;
+
++ export const Primary = meta.story({
+- export const Primary: Story = {
+    // ...current story
++ });
+- };
+```
+
+Note how it is no longer necessary to import any types or manually apply types to the meta or stories. The types are now inferred automatically, thanks to the factory function pattern.
+
+#### 4.1 Reusing story properties
+
+Previously, story properties such as `Story.args` or `Story.parameters` were accessed directly when reusing them in another story. While accessing them like this is still supported, it is deprecated in CSF Factories.
+
+All of the story properties are now contained within a new property called `composed` and should be accessed from that property instead. For instance, `Story.composed.args` or `Story.composed.parameters`.
+
+```diff title="Button.stories.ts"
+// ...rest of file
+
++ export const Primary = meta.story({
+- export const Primary: Story = {
+    args: { primary: true },
++ });
+- };
+
++ export const PrimaryDisabled = meta.story({
+- export const PrimaryDisabled: Story = {
+    args: {
++     ...Primary.composed.args,
+-     ...Primary.args,
+      disabled: true,
+    }
++ });
+- };
+```
+
+<Callout variant="info">
+  The property name "composed" was chosen because the values within are composed from the story, its component meta, and the preview configuration.
+
+  If you wish instead to access only the direct input to the story, you can use `Story.input` instead of `Story.composed`.
```

Comment:
We can go with something like this for clarity
```suggestion
 If you want to access the direct input to the story, you can use `Story.input` instead of `Story.composed`.
```